### DISCUSSION TITLE
how to set default init tst sidebar width  when I restart browser?

### DISCUSSION THREAD

**Discussion by NestorLiao** [piroor/treestyletab#3533] [Q&A]
I am using vertical screen, when I launch firefox, the treestyletab will take over half of the page default.
how could I set the default value?

**Comment by irvinm** [comment]
Roughly speaking, chatgpt has it right ... 

![image](https://github.com/user-attachments/assets/25c31d95-8b09-4063-8242-f04c5104b2ae)
